compiler/wordrecg.nim

```diff
@@ -86,6 +86,8 @@ type
     wInOut, wByCopy, wByRef, wOneWay,
     wBitsize,
 
+    wBenign, wRtl, wNilError, wInl, wCompilerRtl
```

Comment:
Huh? Why are these required?

Reply:
Because we now check all the pragmas and I didn't want to clean up the unused pragmas scattered all around :)

Reply:
Well I don't see why these custom pragmas need to become builtins.

Reply:
I assumed those once were builtins that got removed from the compiler but not from the stdlib, if that's not the case I can either remove them or make them pragma templates.

Reply:
https://github.com/nim-lang/Nim/blob/devel/lib/system/inclrtl.nim